### PR TITLE
feat: add alternative docker bridge

### PR DESCRIPTION
**Pull request by kmkwon94**
Handling issue #606 
Add alternative docker bridge configuration option in `sample.toml` (it would be using as `agent.toml`).
 
If you create new docker bridge with this command `docker network create {new_network}`, the new network is created. Before running the agent, uncomment `container-alternative_bridge` in agent.toml. Then compute session would be created as applying new docker network. This logic would be run in singe-node cluster.

### CODE REVIEW

**Review by kyujin-cho — CHANGES_REQUESTED** [review]
Please define corresponding config scheme on `src/ai/backend/agent/config.py`, just like other values.  Ref: https://github.com/lablup/backend.ai/blob/dcf198c44b07aa348cf719772000c51e0ba5f6ac/src/ai/backend/agent/config.py#L18

File: src/ai/backend/agent/docker/agent.py [hunk 1]
```diff
@@ -434,6 +434,15 @@ async def apply_network(self, cluster_info: ClusterInfo) -> None:
                         },
                     }
                 )
+        else:
+            if "alternate_bridge" in self.local_config["container"]:
```
Comment (adrysn) [inline src/ai/backend/agent/docker/agent.py]: We can merge these lines to reduce the indentation level.

Comment (kmkwon94) [reply src/ai/backend/agent/docker/agent.py]: Thank you. I applied it

File: src/ai/backend/agent/docker/agent.py [hunk 2]
```diff
@@ -434,6 +434,14 @@ async def apply_network(self, cluster_info: ClusterInfo) -> None:
                         },
                     }
                 )
+        elif "alternate-bridge" in self.local_config["container"]:
```
Comment (kyujin-cho) [inline src/ai/backend/agent/docker/agent.py]: Please add non-null check here, as `config["container"]["alternate-bridge"]` value will always exist (either as a string or `None`).
```suggestion
        elif self.local_config["container"].get("alternate-bridge") is not None:
```

**Review by kyujin-cho — CHANGES_REQUESTED** [review]

File: src/ai/backend/agent/docker/agent.py (hunk 2 above)
Comment (kmkwon94) [reply src/ai/backend/agent/docker/agent.py]: Thank you. I applied it.

**Review by kyujin-cho — APPROVED** [review]
LGTM!

**Comment by kyujin-cho** [comment]
closing #606.

**Pull request merged**